src/components/Superfest/SuperfestPage/CTA/MissionCTA.tsx

```diff
@@ -66,12 +73,13 @@ export const MissionCTA = ({ CTAs, variableWeeklyAPY }: MissionCtaProps) => {
             lineHeight={{ xs: '14px', md: '18px' }}
             fontWeight={400}
           >
-            Completing any mission below makes you eligible for OP rewards and
-            XP.
+            {!signature &&
```

Comment:
what is going to be displayed if signature is true?

Reply:
nothing should shown